.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Could we remove this file again?

Reply:
I've added this since I'm working from IntelliJ IDEA. This way I prevent my IDEA setup to be checked in, which I think is preferable?

Reply:
I think this should be in your own global `.gitignore` file instead of this repository. 😉 

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer